debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/walkers/LocalVariableTableIterator.java

```diff
@@ -139,16 +142,21 @@ public LocalVariableTable next() {
 				} else {
 					return null;
 				}
+
+				nameSrp = SelfRelativePointer.cast(localVariableTablePtr);
 				name = J9UTF8Pointer.cast(SelfRelativePointer.cast(localVariableTablePtr).get());
```

Comment:
This can now be simplified:
```
name = J9UTF8Pointer.cast(nameSrp.get());
```
Likewise, for `signature` and `genericSignature`.

Reply:
Done.